utSuiteNavigator.py

```diff
@@ -329,9 +331,14 @@ def select(self, suite_name: str, test_name:str = None, promptWithAnswers:dict =
         return result
 
     def start(self):
-        command = self.config.test.execute
-        #TODO: Handle opkg download and install in the future
-        result = self.framework.start(command)
+        command = self.config.test.execute.command
```

Comment:
unclear why was an issue, since this information should be in `command = self.config.test.execute`